modules/ROOT/pages/portal-vanity-domain.adoc

```diff
@@ -49,7 +49,7 @@ For example:
 * Route `+https://developers.robocop.com/node_modules+` to `+https://anypoint.mulesoft.com/node_modules+`.
 * Route `+https://developers.robocop.com/exchange/api-console+` to `+https://anypoint.mulesoft.com/exchange/api-console+`.
 
-Route `callback` to `+https://anypoint.mulesoft.com/exchange/portals/+**{YOUR_ORGANIZATION_DOMAIN}**/callback`.
+Route `callback` to `+https://anypoint.mulesoft.com/exchange/portals/+**\{YOUR_ORGANIZATION_DOMAIN\}**/callback`.
```

Comment:
Should have a closing + before the ending tick mark.
Unless the asterisks and curly brackets are actually embedded in the URL, it would be much easier to show the URL as `+https://anypoint.mulesoft.com/exchange/portals/YOUR_ORGANIZATION_DOMAIN/callback+`
And lose all the extra characters and their escaping.

Reply:
You're right, that's much better. Fixed.